infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -4,6 +4,18 @@ import * as dynamodb from "@aws-cdk/aws-dynamodb";
 import * as lambdaNodejs from "@aws-cdk/aws-lambda-nodejs";
 import * as cdk from "@aws-cdk/core";
 
+// This is a suboptimal solution to finding the relative directory to the
+// package root. This is necessary because it is possible for this file to be
+// run with the a cwd of either the infrastructure directory or the root of the
+// git repo.
+function packageRoot(): string {
+  const cwd = process.cwd();
+  if (cwd.endsWith("infrastructure")) {
+    return `${cwd}/../packages`;
+  }
+  return `${cwd}/packages`;
+}
```

Comment:
I believe that @zachclark-ccpo and I looked into that. But this is slightly different. _Jest_ isn't scanning these modules and trying to import them or run their tests (at least not in this context, that's for the `npm run test` in `packages/api/`). We're treating these as static files. Basically they're no different to us than if they were PNGs or .TXT files. `rootDir` does not seem to change the working directory for Jest but changes where it scans for files, searches for tests, or the relative path _for other config values_.

This is actually probably good to have in general. I believe it's actually possible to run `npm run cdk` within this directory which means that we'd break in that context too.

There is another pattern for deploying CDK apps where we move the CDK files in "next to" the source files (as we do for the test files today) and that'd be an option as well and might mitigate this confusion but I am not sure whether we actually want to pursue that route.

Because `rootDir` didn't seem to do what we needed and because we really want to get a basic set of tests running, this was identified as the least unpleasant solution. There needs to be a serious refactoring of the copypasta in this file in the near future anyway and I am hoping we can clean this up as part of that.